library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -153,18 +165,47 @@ class Client() {
         return Client(address = address, privateKeyBundleV1 = bundle, apiClient = apiClient)
     }
 
-    fun create(account: SigningKey, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        options: ClientOptions? = null,
+    ): Client {
         val clientOptions = options ?: ClientOptions()
         val apiClient =
             GRPCApiClient(environment = clientOptions.api.env, secure = clientOptions.api.isSecure)
-        return create(account = account, apiClient = apiClient, options = options)
+        return create(
+            account = account,
+            apiClient = apiClient,
+            options = options,
+        )
     }
 
-    fun create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = null): Client {
+    fun create(
+        account: SigningKey,
+        apiClient: ApiClient,
+        options: ClientOptions? = null,
+    ): Client {
         return runBlocking {
             try {
                 val privateKeyBundleV1 = loadOrCreateKeys(account, apiClient, options)
-                val client = Client(account.address, privateKeyBundleV1, apiClient)
+                val libXMTPClient: FfiXmtpClient? =
+                    if (options != null && options.enableLibXmtpV3 && options.appContext != null) {
+                        val dbDir = File(options.appContext.filesDir.absolutePath, "xmtp_db")
+                        dbDir.mkdir()
+                        val dbPath: String = dbDir.absolutePath + "/xmtp-${account.getAddress()}.db3"
```

Comment:
I'm naming the db after the account address so that there isn't collisions if multiple addresses log in on the same device.

Reply:
👍

Reply:
Maybe `$env-$address` so that you can switch between dev/local/prod in the same app